sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -424,3 +424,25 @@ function buildRawAuthorizationRule(authorizationRule: AuthorizationRule): any {
   };
   return rawAuthorizationRule;
 }
+
+/**
+ * @ignore
+ * Helper utility to extract the relative path from given URL,
+ * or undefined / empty if not existing.
+ * Valid relative paths used in the ATOM based management API are in format
+ *     - `<queue-name>`
+ *     - `<topic-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>/Rules/<rule-name>`
+ * @param url
+ */
+export function getRelativePathOrUndefined(url: string): string | undefined {
+  let stringValue = getStringOrUndefined(url);
+  if (stringValue == undefined) {
+    return undefined;
+  }
+
+  stringValue = stringValue.substring(9);
+  const indexOfFirstSlash = stringValue.indexOf("/");
+  return stringValue.substring(indexOfFirstSlash + 1);
```

Comment:
The docs on `forwardTo` and `forwardDeadletterMessagesTo`  describe them to be relative paths. Then, why are we trying to support the case where the user provides absolute urls?

Even if we want to support the case where user provides absolute urls, can we not use the urls directly instead of converting them to relative path here only to be suffixed to `https://${this.endpoint}` later?

Also, when the user provides say just the queue name for `forwardTo` (because thats what the docs ask them to do), arent we ending up treating the queue name as a url here and end up substringing it here and losing the information?